lib/src/controls/navigation/tab_view.dart

```diff
@@ -1062,6 +1076,15 @@ class _TabPainter extends CustomPainter {
         size.width + radius,
         size.height,
       );
+
+    if (outlineColor != null) {
+      final outlinePaint = Paint()
+        ..color = outlineColor!
+        ..style = PaintingStyle.stroke
+        ..strokeWidth = 2.0;
+
+      canvas.drawPath(path, outlinePaint);
+    }
```

Comment:
Can you show how this would look like? IIRC, this is not part of the official documentation or implementation.

Reply:
<img width="504" alt="dark" src="https://github.com/bdlukaa/fluent_ui/assets/78206446/e67bbf00-d799-446b-b8ac-8aa04d8a6609">
<img width="505" alt="light" src="https://github.com/bdlukaa/fluent_ui/assets/78206446/6ad69c9f-db63-418a-9220-6f59f63f6240">